src/harness/fourslash.ts

```diff
@@ -704,13 +704,55 @@ module FourSlash {
             }
         }
 
-        public verifyCompletionListDoesNotContain(symbol: string) {
+        /**
+         * Verfiy that the completion list does NOT contain the given symbol. If text, or documentation, or kind are provided,
+         * the list contains the symbol all given parameters must matched. When any parameter is omitted, the parameters is ignored during comparison.
+         */
+        public verifyCompletionListDoesNotContain(symbol: string, expectedText?: string, expectedDocumentation?: string, expectedKind?: string) {

```

Comment:
Why do any of these parameters help? You're interested n the lack of `symbol` in the list, so if `symbol` isn't in the list, then why do you care about any of the other parameters.